js/pages/SeatPage.ts

```diff
@@ -157,7 +157,7 @@ export class SeatPage extends PageBase {
         });
     }
     public calculateWidth() {
-        if (appConfig.ui.usePortraitModeOnly) {
+        if (settings.orientation() === "portrait") {
```

Comment:
This expression could be part of method `isTargetOrientation` `OrientationService`. I want to prevent typo errors. I'm fine if type of `settings.orientation()` would be not `string` but `"portrait" | "landscape"` to allow only two string values.

Reply:
Maybe make sense to create function `isPortraitOrientationRequired` in SeatPage and TablesPage for consistency and hide implementation details. Also this is make intent clearer

Reply:
yeap this makes sense, i'll do it.

Reply:
Maybe i should make some independent function that will do this stuff for both seat and tables pages, as  we duplicate some parts on this pages?